internal/kernel/marketdata/weather_test.go

```
@@ -0,0 +1,91 @@
+package marketdata
+
+import (
+	"context"
+	"net/http"
+	"testing"
+)
+
+func i64(v int64) *int64 { return &v }
+func b(v bool) *bool     { return &v }
```

golangci-lint (2.12.2)

[error] func b is unused

(unused)